plugins/base/src/main/kotlin/signatures/KotlinSignatureProvider.kt

```diff
@@ -128,34 +129,36 @@ class KotlinSignatureProvider(ctcc: CommentsToContentConverter, logger: DokkaLog
                             else -> c.modifier[sourceSet]?.name?.let { "$it " } ?: ""
                         }
                     else
-                        ""
+                        "", styles = mainStyles + TokenStyle.Keyword
                 )
             }
             when (c) {
                 is DClass -> {
                     processExtraModifiers(c)
-                    text("class ")
+                    text("class ", styles = mainStyles + TokenStyle.Keyword)
                 }
                 is DInterface -> {
                     processExtraModifiers(c)
-                    text("interface ")
+                    text("interface ", styles = mainStyles + TokenStyle.Keyword)
                 }
                 is DEnum -> {
                     processExtraModifiers(c)
-                    text("enum ")
+                    text("enum ", styles = mainStyles + TokenStyle.Keyword)
                 }
                 is DObject -> {
                     processExtraModifiers(c)
-                    text("object ")
+                    text("object ", styles = mainStyles + TokenStyle.Keyword)
                 }
                 is DAnnotation -> {
                     processExtraModifiers(c)
-                    text("annotation class ")
+                    text("annotation class ", styles = mainStyles + TokenStyle.Keyword)
                 }
             }
             link(c.name!!, c.dri)
             if (c is WithGenerics) {
-                list(c.generics, prefix = "<", suffix = ">") {
+                list(c.generics, prefix = "<", suffix = ">",
+                    separatorStyles = mainStyles + TokenStyle.Punctuation,
+                    borderStyles = mainStyles + TokenStyle.Operator) {
```

Comment:
i'd name it `surroundingCharactersStyle` or any other name related to `surrounding`